### PR TITLE
add popup_bidi plugin

### PR DESCRIPTION
**Pull request by hosseinprince**
- [x] I'm the package's author and/or maintainer.
- [ ] I have have read [the docs][1].
- [ ] I have tagged a release with a [semver][2] version number.
- [ ] My package repo has a description and a README describing what it's for and how to use it.
- [ ] My package doesn't add context menu entries. *
- [ ] My package doesn't add key bindings. **
- [ ] Any commands are available via the command palette.
- [ ] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [ ] If my package is a syntax it doesn't also add a color scheme. ***
- [ ] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is ...

There are no packages like it in Package Control.

My package is similar to ... However it should still be added because ...

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

File: repository/p.json [hunk 1]
```diff
@@ -1910,6 +1910,17 @@
 				}
 			]
 		},
+		{
+			"name": "popup_bidi",
+			"details": "https://github.com/hosseinprince/popup_bidi",
+			"labels": ["rtl","bidi","persian","arabic"],
+			"releases": [
+				{
+					"sublime_text": "*",
```
[the remark this answers is not in the document]
Comment (hosseinprince) [reply repository/p.json]: udpate that

**Comment by braver** [comment]
According to your checklis you haven't read the readme, not tagged a release. Please complete the checklist (for reals) and tell us a little bit about your package.

**Comment by braver** [comment]
- Maybe "Bidirectional text popup" would be a more descriptive name.
- Your readme says "Bidirectional popup support for Sublime Text 3", but ST has been at version 4 for a while and I assume that is supported as well.

Please re-check the checklist and respond to this feedback to continue the review.

**Comment by braver** [comment]
Please respond to the feedback to continue the review.

**Comment by braver** [comment]
Closing due to lack of activity. We can always re-open and continue the review when you're ready for it.

**Pull request closed without merging**